plists/MigrationMap.plist

```diff
@@ -718,6 +718,18 @@
       <string>xterm</string>
     </dict>
     <dict>
+        <key>Prefix</key>
```

Comment:
Revert changes to this file. It's only needed for prefs that existed before iTerm2 existed.